File: Tournonment/BazgashtAzBoostanS1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String input1 = scanner.nextLine();
        String input2 = scanner.next();

        if (input1.charAt(0) >= input2.charAt(0))
            System.out.print("Left");
        else
            System.out.print("Right");
    }
}

// </Ctrl+Z>

File: Tournonment/BazgashtAzBoostanS1.py
import sys
class Main(object) :
    @staticmethod
    def main( args) :
        scanner =  "Python-inputs"
        input1 = input()
        input2 = input()
        if (input1[0] >= input2[0]) :
            print("Left", end ="")
        else :
            print("Right", end ="")


if __name__=="__main__":
    Main.main(sys.argv)
